### ISSUE TITLE
Fine-grained classification train/test split mistake?

### ISSUE THREAD

**Issue by iamhankai** [bogger/caffe-multigpu#2]
I have seen that in the prototxt file "https://github.com/bogger/caffe-multigpu/blob/multigpu/models/finetune_web_car/googlenet/train_val_googlenet.prototxt". 
There are 16079 samples in "/home/ljyang/work/caffe/caffe-multigpu/models/finetune_web_car/lists/classification_train", and 28097 sample in test set. But in your paper 70% images are for training and 30% are for testing.

What's wrong?
